sectionsUtilityTool/src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionUtilityToolFilter.java

```diff
@@ -0,0 +1,171 @@
+package edu.umich.ctools.sectionsUtilityTool;
+
+import java.io.IOException;
+import java.util.Hashtable;
+import java.util.Properties;
+
+import javax.naming.Context;
+import javax.naming.NamingEnumeration;
+import javax.naming.NamingException;
+import javax.naming.directory.Attribute;
+import javax.naming.directory.DirContext;
+import javax.naming.directory.InitialDirContext;
+import javax.naming.directory.SearchControls;
+import javax.naming.directory.SearchResult;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import edu.umich.its.lti.utils.PropertiesUtilities;
+
+
+
+public class SectionUtilityToolFilter implements Filter {
```

Comment:
These values are private static final.  Any value that might change should be a variable that can be overridden from the properties file.  The provider_url in particular might need to be changed.  E.g. it would be a very reasonable request from ITS to use a test LDAP server for test installations and a production LDAP server for a production installation.